File: Sources 4.5.2/Gizmox.WebGUI.Forms.CompanionKit/Controls/ActiveXBox/ApplicationScenario/ExampleApplicationPage.cs
using System;
using System.Text;
using System.Data;
using System.Drawing;
using System.ComponentModel;
using System.Collections.Generic;

using Gizmox.WebGUI.Forms;
using Gizmox.WebGUI.Common;

namespace CompanionKit.Controls.ActiveXBox.ApplicationScenario
{
    public partial class ExampleApplicationPage : UserControl
    {
        public ExampleApplicationPage()
        {
            InitializeComponent();
        }

        /// <summary>
        /// Handles Load event of the example' UserControl
        /// Set parameters for the demonstrated ActiveXBox
        /// </summary>
        private void ExampleApplicationPage_Load(object sender, EventArgs e)
        {
            // Set parameters for the AciveXBox             
            this.demoActiveXBox.AddParameter("autoStart", "False");
            this.demoActiveXBox.AddParameter("URL", "../../../../../../../Resources/video/Example.wpl");
            this.demoActiveXBox.AddParameter("enabled", "True");
            this.demoActiveXBox.AddParameter("balance", "0");
            this.demoActiveXBox.AddParameter("currentPosition", "0");
            this.demoActiveXBox.AddParameter("enableContextMenu", "True");
            this.demoActiveXBox.AddParameter("fullScreen", "False");
            this.demoActiveXBox.AddParameter("mute", "False");
            this.demoActiveXBox.AddParameter("rate", "1");
            this.demoActiveXBox.AddParameter("stretchToFit", "False");
            this.demoActiveXBox.AddParameter("uiMode", "full");
        }
    }
}

File: Sources 4.5.2/Gizmox.WebGUI.Forms.CompanionKit/Controls/ActiveXBox/ApplicationScenario/ExampleApplicationPage.vb
Namespace CompanionKit.Controls.ActiveXBox.ApplicationScenario

	Public Class ExampleApplicationPage

		Public Sub New()

			' This call is required by the Visual WebGui UserControl Designer.
			InitializeComponent()

	        

		End Sub

        ''' <summary>
        ''' Handles Load event of the example' UserControl
        ''' Set parameters for the demonstrated ActiveXBox
        ''' </summary>
        Private Sub ExampleApplicationPage_Load(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles MyBase.Load
            '  Set parameters for the AciveXBox  
            Me.demoActiveXBox.AddParameter("autoStart", "False")
            Me.demoActiveXBox.AddParameter("URL", "../../../../../../Resources/video/Example.wpl")
            Me.demoActiveXBox.AddParameter("enabled", "True")
            Me.demoActiveXBox.AddParameter("balance", "0")
            Me.demoActiveXBox.AddParameter("currentPosition", "0")
            Me.demoActiveXBox.AddParameter("enableContextMenu", "True")
            Me.demoActiveXBox.AddParameter("fullScreen", "False")
            Me.demoActiveXBox.AddParameter("mute", "False")
            Me.demoActiveXBox.AddParameter("rate", "1")
            Me.demoActiveXBox.AddParameter("stretchToFit", "False")
            Me.demoActiveXBox.AddParameter("uiMode", "full")
        End Sub
    End Class

End Namespace